index.d.ts

```diff
@@ -461,26 +461,41 @@ declare module '@dramancompany/remember-ui' {
 
   export const Textarea: React.FC<TextareaProps>;
 
+  export type TippyPlaceMentType =
+    | 'top'
+    | 'top-start'
+    | 'top-end'
+    | 'right'
+    | 'right-start'
+    | 'right-end'
+    | 'bottom'
+    | 'bottom-start'
+    | 'bottom-end'
+    | 'left'
+    | 'left-start'
+    | 'left-end';
   export interface PopoverProps {
     content: ReactNode;
     onPopoverOpenChange?: Function;
     className?: string;
     customElement?: ReactNode;
-    size?: string;
+    size?: 'small' | 'large';
     onOpen?: Function;
     distance?: number;
-    trigger?: string;
-    placement?: string;
+    trigger?: 'mouseenter' | 'click' | 'focus';
+    placement?: TippyPlaceMentType;
```

Comment:
```suggestion
    placement?: TippyPlacementType;
```
- camelCase 에 맞지 않는거 같습니다!
- 그리고 이 부분 타입이 사실 잘 따라가보면 Tippy가 의존하고 있는 popper의 placement 타입을 쓰면 된다는 사실을 알고 있는데요, Tippy 타입 중 TippyProps에서 extends 하고 있어서 tippy 관련 props는 직접 정의하기보다 이걸 갖다 쓰도록 하는게 좋을거 같습니다

Reply:
반영하였습니다!